lib/oppen.rb

```diff
@@ -38,54 +39,105 @@ def self.print(config: Config.oppen, space: ' ',
 
   # Config.
   class Config
-    # IndentAnchor.
+    # The different ways of handling the indentation of nested groups.
+    # (cf. test/indent_anchor_test.rb).
     #
-    # ON_BREAK => anchor on break position (as in Oppen's original paper)
-    # ON_BEGIN => anchor on begin block position
+    # @example
+    #   config = Oppen::Config.new(indent_anchor: END_OF_PREVIOUS_LINE)
+    #   out = Oppen::Wadler.new config:, width: 13
+    #   out.text 'And she said:'
+    #   out.group(4) {
+    #     out.group(4) {
+    #       out.break
+    #       out.text 'Hello, World!'
+    #     }
+    #   }
+    #   out.output
+    #
+    #   # Returns the following:
+    #   #
+    #   # And she said:
+    #   #                  Hello, World!
```

Comment:
```suggestion
    #   puts out.output
    #
    #   # =>
    #   # And she said:
    #   #                  Hello, World!
```

Same for other examples.